jenkins/bootstrap_test.py

```diff
@@ -982,7 +984,12 @@ def testOnlyJobs(self):
             self.assertTrue(os.access(job_path, os.X_OK|os.R_OK), job_path)
 
     def testAllProjectAreUnique(self):
-        projects = collections.defaultdict(list)
+        allowed_list = {  # TODO(fejta): remove these
```

Comment:
Oh sorry I was thinking of you want to remove this TODO rather than remove the jobs..